test/integration/dialects/postgres/query-generator.test.js

```diff
@@ -220,6 +220,13 @@ if (dialect.match(/^postgres/)) {
         }
       ],
 
+      dropSchema: [
+        {
+          arguments: ['mySchema'],
+          expectation: 'DROP SCHEMA IF EXISTS mySchema CASCADE;'
+        },
+      ],
```

Comment:
SQL unit tests should be in test/unit/sql/*
the SQL tests in test/integration are an outdated thing i'm afraid.